pombola/images/migrations/0002_auto_20160229_1343.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you give this migration a more descriptive name than "auto_20160229_1343"? :)